### PR TITLE
feat(Persona): add `renderButton` prop

### PR DESCRIPTION
**Pull request by ogonkov**
Added new `renderButton` prop, that allows to override default button. Default button is now exposed as `<Persona.Button/>` for convenience.

### CODE REVIEW

File: src/components/PersonaWrap/PersonaWrap.tsx [hunk 1]
```diff
@@ -1,46 +1,33 @@
...
     onClick,
     onClose,
+    renderButton,
     className,
     avatar,
     children,
 }) => {
     const clickable = Boolean(onClick);
-    const closeable = Boolean(onClose);
+    const closeable = Boolean(onClose || renderButton);
```
Comment (DaffPunks) [inline src/components/PersonaWrap/PersonaWrap.tsx:22]: Мне кажется теперь у этой переменной другой смысл. Не `closeable`, а какой-нибудь `withAction`

**Review by DaffPunks — APPROVED** [review]

File: src/components/PersonaWrap/PersonaWrap.tsx (hunk 1 above)
Comment (ogonkov) [reply src/components/PersonaWrap/PersonaWrap.tsx:22]: It's alvays was like that)

File: src/components/Persona/Persona.tsx [hunk 2]
```diff
@@ -47,13 +30,18 @@ export const Persona: FC<PersonaProps> = ({
             break;
     }
 
+    const renderCloseButton = useCallback(
```
Comment (amje) [inline src/components/Persona/Persona.tsx:33]: This button should always be close button. Let's not add a custom renderer here

**Review by amje — CHANGES_REQUESTED** [review]

**Comment by ogonkov** [comment]
I guess not

**Pull request closed without merging**